Sources/TuistKit/Graph/GraphNode.swift

```diff
@@ -1,5 +1,6 @@
 import Basic
 import Foundation
+import enum ProjectDescription.SDKStatus
```

Comment:
We'll soon be decoupling the generator code to its own module (#205) `TuistGenerator` such that it's independent from `ProjectDescription` - This is an interesting technique though which I haven't previously considered.  

Currently we have replicas of the types within the `TuistKit`.